python/src/internal/ChdDictionary.py

```diff
@@ -0,0 +1,173 @@
+import hashlib
+from collections import defaultdict
+import zlib
+from typing import Any, Tuple
+from typing import Generic, TypeVar, Dict
+
+
+from python.src import GmodNode
+
+TValue = TypeVar('TValue')
+
+class ChdDictionary(Generic[TValue]):
```

Comment:
Denne klassen er det martin som lagde i C# for å forbedre perfomance. Den finnes ikke i TS, så ville sjekket med og uten når du begynner å benchmarke

Reply:
Denne ble fjernet